code/datums/voice_announcements.dm

```diff
@@ -13,7 +12,7 @@ GLOBAL_LIST_EMPTY(voice_announce_list)
 /datum/voice_announce/New(client/client)
 	. = ..()
 	src.client = client
-	id = "[client.ckey]_[sha1("[world.time]-[world.realtime]-[world.timeofday]-[rand()]")]"
+	id = "[client.ckey]_[sha1(GUID())]"
```

Comment:
Again, im not sure if this is meant to be secured or not, but if it is, GUIDs are not secure either
https://datatracker.ietf.org/doc/html/rfc4122#section-6
![img](https://i.imgur.com/qIq8uIJ.png)